src/FruitCounter.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+function FruitCounter({emoji, fruitTitle, fruitName, setFruitCount, fruitCount}) {
+    return (
+        <div className={fruitCount > 0 ? "counterCardFilled" : "counterCard"}>
+            <h2 className="fruitTitle">{emoji} {fruitTitle} {emoji}</h2>
+            <div className="counterButtons">
+                <button className={fruitCount > 0 ? "fruitButtonFilled" : "fruitButton"}
+                    type="button"
+                    onClick={() => setFruitCount(fruitCount - 1)}
```

Comment:
Nu zal telkens wanneer `fruitCount` veranderd een onnodige rerender van deze functie plaatsvinden. Wat je zou kunnen doen is de huidige waarde gebruiken die je vanuit de setFruitCount functie al krijgt:
```suggestion
                    onClick={() => setFruitCount(currentValue => currentValue - 1)}
```